operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,504 @@
+# Rack Resiliency (RR)
+
+* [Introduction](#introduction)
+* [RR solution overview](#rr-solution-overview)
+* [Steps to achieve RR](#steps-to-achieve-RR)
+* [Verification of RR](#verification-of-RR)
+* [Manage RR via CLI](#Manage-RR-via-CLI)
+* [Manage Critical Services](#manage-critical-services)
+* [Glossary](#glossary)
+
+## Introduction
+
+HPE Cray EX systems are engineered to ensure services critical to the execution of user jobs remain resilient, even if management nodes (Master, Worker, and Storage nodes) fail, provided that the minimum quorum of nodes required for high availability (HA) is maintained.

```

Comment:
All of these long lines, in this and other files, need to be broken up. It's going to make the review process much longer and more tedious when so many different changes may be suggested for just a single, very long line.